source/enc-h264.cpp

```diff
@@ -63,7 +63,8 @@ const char* Plugin::Interface::H264Interface::get_name(void*) noexcept
 	return "H264/AVC Encoder (" PLUGIN_NAME ")";
 }
 
-void* Plugin::Interface::H264Interface::create(obs_data_t* settings, obs_encoder_t* encoder) noexcept try {
+void* Plugin::Interface::H264Interface::create(obs_data_t* settings, obs_encoder_t* encoder) noexcept
+try {
```

Comment:
Can you rebase your changes onto the clang commit?

Reply:
I'm on latest everything, and this was the result of clang-format.

Reply:
Ah my bad, i haven't actually merged this into this repo. I'll merge this first, then merge my changes.